<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCommSettings</class>
 <widget class="QDialog" name="DlgCommSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>НАСТРОЙКИ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_26">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Принтер на апликатора</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Порт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboLabelPortName"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Скорост</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboLabelBaud"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>данни</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="rbLabelDataBits8">
               <property name="text">
                <string>8</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbLabelDataBits7">
               <property name="text">
                <string>7</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>четност</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="rbLabelParityNone">
               <property name="text">
                <string>none</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbLabelParityOdd">
               <property name="text">
                <string>odd</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbLabelParityEven">
               <property name="text">
                <string>even</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>стоп</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_20">
             <item>
              <widget class="QRadioButton" name="rbLabelStopBits1">
               <property name="text">
                <string>1</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbLabelStopBits2">
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>14</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Модел</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboLabelPrinterType">
            <item>
             <property name="text">
              <string>SATO</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Принтер на сумарен етикет</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Порт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboTotalPortName"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Скорост</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboTotalBaud"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>данни</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QRadioButton" name="rbTotalDataBits8">
               <property name="text">
                <string>8</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbTotalDataBits7">
               <property name="text">
                <string>7</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>четност</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QRadioButton" name="rbTotalParityNone">
               <property name="text">
                <string>none</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbTotalParityOdd">
               <property name="text">
                <string>odd</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbTotalParityEven">
               <property name="text">
                <string>even</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>стоп</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_21">
             <item>
              <widget class="QRadioButton" name="rbTotalStopBits1">
               <property name="text">
                <string>1</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbTotalStopBits2">
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>14</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Модел</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboTotalLabelPrinterType">
            <item>
             <property name="text">
              <string>Citizen</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="title">
        <string>Везна</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Порт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboScalePortName"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Скорост</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboScaleBaud"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_10">
            <property name="title">
             <string>данни</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QRadioButton" name="rbScaleDataBits8">
               <property name="text">
                <string>8</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbScaleDataBits7">
               <property name="text">
                <string>7</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_11">
            <property name="title">
             <string>четност</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QRadioButton" name="rbScaleParityNone">
               <property name="text">
                <string>none</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbScaleParityOdd">
               <property name="text">
                <string>odd</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbScaleParityEven">
               <property name="text">
                <string>even</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_12">
            <property name="title">
             <string>стоп</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_22">
             <item>
              <widget class="QRadioButton" name="rbScaleStopBits1">
               <property name="text">
                <string>1</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbScaleStopBits2">
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>14</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Модел</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboScaleType"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QGroupBox" name="groupBox_14">
       <property name="title">
        <string>Измерване на тегло</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_24">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Цикли на осредняване</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Защитно време (ms)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Праг на сработване (g)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Праг на освобождаване (g)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_23">
            <item>
             <widget class="QLineEdit" name="le_averagingCicles"/>
            </item>
            <item>
             <widget class="QLineEdit" name="le_mesureDelay"/>
            </item>
            <item>
             <widget class="QLineEdit" name="le_threshold"/>
            </item>
            <item>
             <widget class="QLineEdit" name="le_lowThreshold"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_25">
       <item>
        <widget class="QGroupBox" name="groupPrintTotal">
         <property name="title">
          <string>Печат на етикет тотал</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Брой етикети</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_totalCount">
              <property name="maximum">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>ПОТВЪРДИ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>ОТКАЖИ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgCommSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
